x-pack/plugins/cloud/public/types.ts

```diff
@@ -72,6 +72,11 @@ export interface CloudStart {
      * Will always be present if `isServerlessEnabled` is `true`
      */
     projectId?: string;
+    /**
+     * The serverless project name.
+     * Will always be present if `isServerlessEnabled` is `true`
+     */
+    projectName?: string;
```

Comment:
Technically, will only be present once the PR to inject the info from the controller is merged, but I don't think we need to have this in the comments.